src/github.ts

```diff
@@ -86,6 +86,7 @@ export interface GitHubReleasePR {
   number: number;
   version: string;
   sha: string;
+  normalizedVersion: string;
```

Comment:
not a request to change the approach, but curious why you opted to track both `version` and `normalizedVersion`.

Reply:
Ah, good question! It looks like a few pieces in `github-release.ts` are still expecting tag names. But it's not used anywhere else, so it makes sense to me to just shift the normalized one back to just `version` and then make the few places add a `v`.

Reply:
Looks like a few tests were failing:
```
  1) GitHubRelease
       createRelease
         creates and labels release on GitHub:
     Uncaught Error: Different value of snapshot "GitHubRelease createRelease creates and labels release on GitHub"
{
  name: "foo v1.0.3" => "foo 1.0.3"
  tag_name: "v1.0.3" => "1.0.3"
}
```
But I think it should be pretty easy to just fix up the places in github-release.ts that need this. Then I can get rid of the normalized member and make `version` the normalized one.